### PR TITLE
Added DHCP config generated from Ralph

### PR DESCRIPTION
**Pull request by ar4s**
Related to #2276

### CODE REVIEW

**Comment by szok** [comment]
:ok:

**Comment by vi4m** [comment]
When are we going to separate it into separate package(inside the same repo but still different directory) ?

File: src/ralph/assets/models/components.py [hunk 1]
```diff
@@ -77,3 +84,25 @@ class GenericComponent(Component):
     class Meta:
         verbose_name = _('generic component')
         verbose_name_plural = _('generic components')
+
+
+class Ethernet(Component):
+    label = NullableCharField(
+        verbose_name=_('name'), max_length=255, blank=True, null=True
+    )
+    mac = models.CharField(
```
Comment (szok) [inline src/ralph/assets/models/components.py:93]: Why not used: NullableCharFieldMixin ?

**Comment by mkurek** [comment]
:ok:

**Pull request merged**